modal.css

```diff
@@ -0,0 +1,312 @@
+  justify-content: center;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.7);
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  backdrop-filter: blur(10px);
+}
+
+@media (min-width: 375px) {
+  #modal-box {
+    background-color: rgb(255, 255, 255);
+    height: 718px;
+    width: 100%;
+    max-width: 343px;
+    margin-top: 100px;
+    top: 60px;
+    left: 16px;
+    box-shadow: 0 0 rgba(241, 90, 34, 0.8);
+    transform: scale(1);
+    padding: 16px;
+    z-index: 9999999999999999;
+    transition: 200ms ease-in-out;
+  }
+
+  #modal-box.active {
+    transform: scale(1);
+  }
+
+  #modal-wrapper.active {
+    display: flex;
+  }
+
+  #modal-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  #modal-box #close-modal {
+    cursor: pointer;
+    border: none;
+    outline: none;
+    background: none;
+    font-size: 2.25rem;
+    font-weight: bold;
+    color: #333;
+  }
+
+  #modal-header h2 {
+    width: 259px;
+    height: 82px;
+    font-family: 'Roboto', sans-serif;
+    font-weight: bold;
+    font-size: 30px;
+    line-height: 44px;
+    color: #172b4d;
+    margin-top: 0;
+  }
+
+  .close-modal {
+    margin-top: -65px;
+    color: rgb(247, 243, 243);
+  }
+
+  .popup-lang {
+    display: flex;
+    width: 90px;
+    height: 36px;
+    gap: 10px;
+    margin-left: -40px;
+    list-style-type: none;
+    margin-top: -10px;
+    margin-bottom: 50px;
+  }
+
+  .feature-image {
+    width: 311px;
+    height: 220px;
+    margin-top: -25px;
+    left: 24px;
+  }
+
+  .side-info p {
+    width: 311px;
+    height: 220px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 20px;
+    color: #344563;
+    margin-top: 0;
+  }
+
+  .popup-lang a {
+    text-decoration: none;
+    border: 1px solid #979493;
+    padding: 10px 12px;
+    cursor: pointer;
+    font-family: 'Poppins', sans-serif;
+    font-weight: bold;
+    font-size: 13px;
+    color: #3c3a39;
+    line-height: 20px;
+    width: 39px;
+    height: 20px;
+  }
+
+  .out-links a {
+    width: 220px;
+    margin-top: 15px;
+    left: 708px;
+    padding-bottom: 10px;
+    margin-right: 12px;
+    background-color: #f55800;
+    text-decoration: none;
+    color: white;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 3%;
+    text-align: center;
+  }
+
+  .popup-lang a:hover {
+    background-color: #f55800;
+    color: white;
+  }
+
+  .out-links a:hover {
+    background-color: white;
+    color: #f55800;
+  }
+
+  .out-links {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    right: 50%;
+  }
+
+  .out-links img {
+    padding-left: 10px;
+    padding-top: 20px;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  #modal-box {
+    background-color: rgb(255, 255, 255);
+    width: 70%;
+    height: auto;
+    max-height: 75%;
```

Comment:
- [ ] Great job incorporating the changes suggested by the previous reviewer. However, while implementing those changes, it appears that a new bug has been introduced into your code. The bug causes your `model` to become less responsive, resulting in content overflowing and potentially affecting the user experience. To `fix` this issue kindly consider the below suggestions:
  - [ ] Consider removing the `width: 70%` because it's causing your model content to overflow from the right. This is because the width of the model content is greater than that of its container which you have set at `70%`.
  - [ ] For the same reason mentioned above about content overflowing kindly consider removing the `max-height: 75%`. See below for more details: 
  
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="1192" alt="Screenshot 2023-07-01 at 10 03 34" src="https://user-images.githubusercontent.com/99394374/250252568-ce943f05-b235-4cde-9411-f2b648c0e39c.png">
</td>
<td><img width="1192" alt="Screenshot 2023-07-01 at 10 03 34" src="https://github.com/anyars-encarta/anyars-encarta.github.io/assets/107192076/86b02f5c-1a71-4f01-9656-69673079ae34"></td>
</tr>
</table>